ts/main.ts

```diff
@@ -37,4 +45,88 @@ $idJournalEntry.addEventListener('submit', (event: Event) => {
   writeData();
   $image.src = 'images/placeholder-image-square.jpg';
   $idJournalEntry.reset();
+  const li = renderEntry(formObject);
+  const $tagFirstLi = document.querySelector('li');
+  $classJournalEntries?.insertBefore(li, $tagFirstLi);
```

Comment:
What if you don't have any `li`'s to begin with? I would just `prepend` directly to the `ul`
```suggestion
  const $ul= document.querySelector('ul');
  $ul.prepend(li);
```